src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php

```diff
@@ -101,9 +101,14 @@ public function onKernelRequest(GetResponseEvent $event)
             }
         }
 
-        $token = urldecode($request->query->get('_token', false));
+        $token = false;
+        if ($request->query->has('_token')) {
+            $token = new CsrfToken($this->username, $request->query->get('_token'));
+        } elseif ($request->query->has('form')) {
```

Comment:
This is our generic form name and also the same as Symfony form when form are created without a form factory / form builder: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Form/FormHandler.php#L85
We currently don't have any choice because form are badly created, and csrf tokens are badly used :sweat_smile: 
Must be fixed in 1.7.8.0